sklearn/model_selection/_search.py

```diff
@@ -1066,11 +1152,19 @@ class RandomizedSearchCV(BaseSearchCV):
         Refer :ref:`User Guide <cross_validation>` for the various
         cross-validation strategies that can be used here.
 
-    refit : boolean, default=True
+    refit : boolean, or string default=True
```

Comment:
Reading this again, I'm not that convinced that is the right choice any more. Why not no refitting by default in the multiple metric case? People can always refit by hand using the best parameters for any of the metrics.